example/src/Tests.tsx

```diff
@@ -100,9 +100,9 @@ export const Tests: React.FC = () => {
   return (
     <View>
       <Text>{allTestsPassed ? 'Tests passed' : 'Tests failed'}</Text>
-      {tests.map((result) => {
+      {tests.map((result, index) => {
         return (
-          <View key={performance.now()}>
+          <View key={index}>
```

Comment:
this afaik can become a problem, because in a follow up run of the rendering it afaik might not update the entry?

Reply:
I would just keep `performance.now()` because it's always unique - or generate a unique id somehow without pulling in a package